index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        <title>Apple</title>
+        <link rel="stylesheet" href="css/style.css" type="text/css" />
+        <script src="https://kit.fontawesome.com/1b0418f0c5.js" crossorigin="anonymous"></script>
+    </head>
+
+    <body>
+        <main class="main">
+            <nav class="nav-container">
+                <div class="first"><img class="apple" src="images-apple/logo-apple.png" alt="apple" /></div>
+                <div>
+                    <p>Store</p>
+                </div>
+                <div>
+                    <p>Mac</p>
+                </div>
+                <div>
+                    <p>iPod</p>
+                </div>
+                <div>
+                    <p>iPhone</p>
+                </div>
+                <div>
+                    <p>iPad</p>
+                </div>
+                <div>
+                    <p>iTunes</p>
+                </div>
+                <div>
+                    <p>Support</p>
+                </div>
+                <div class="last">
+                    <span class="search-box"> <img class="search" src="images-apple/search1.PNG" alt="search"/> </span>
+                </div>
+            </nav>
+            <div class="text">
+                <p class="what">
+                    What will your verse be?
+                </p>
+                <p class="everyone">
+                    Everyone has something to share. See how people are using <br />
+                    iPad in their unique ways to add to the world's story.
+                </p>
+                <div class="verse">
+                    <p class="watch">Watch the "Your Verse" film and explore the stories</p>
+                    <img src="images-apple/play-button-overlay-png.png" alt="play" />
+                </div>
+            </div>
+        </main>
+        <section></section>
```

Comment:
**Mandatory**
- This section must contain four elements which are separate links (to other articles or `#`)

Reply:
Thanks for the review.